src/mscorlib/shared/System/Threading/Tasks/IValueTaskSource.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Threading.Tasks
```

Comment:
I thought we agreed to put the source APIs in a subnamespace?

Reply:
See https://github.com/dotnet/coreclr/pull/16618#issuecomment-369291758.

We discussed it.  There was no conclusion.  I'm happy for you to make a call on if and what it should be.  My default is to leave them where they are.

Reply:
I also don't feel super strongly. @terrajobst?